src/pages/youthboard/surveys/index.tsx

```
@@ -0,0 +1,132 @@
+
+
+        </Box>
+
+        <Box>
+        {value === 1 && (
+          <Box padding={"15px"} sx={{
+            background: '#FBF4E4',
+          }}>
+            <Grid container spacing={2}>
+              {surveysData.map((survey, index) => (
+                <Grid item xs={12} sm={12} md={6} lg={4}> 
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)